local/bin/py/build/configurations/pull_config_preview.yaml

```
@@ -1,6 +1,6 @@
 ---
 - config:
-    cache_enabled: true
+    cache_enabled: false
```

apply suggestion before merge 🙇 
```suggestion
    cache_enabled: true
```